core/scripts/cre/environment/environment/environment.go

```diff
@@ -680,16 +712,27 @@ func StartCLIEnvironment(
 		readContractBinaryName = "readcontract"
 	}
 
+	httpActionBinaryName := filepath.Base(in.ExtraCapabilities.HTTPActionBinaryPath)
+	if withPluginsDockerImageFlag != "" {
+		httpActionBinaryName = "http_action"
+	}
+	httpTriggerBinaryName := filepath.Base(in.ExtraCapabilities.HTTPTriggerBinaryPath)
+	if withPluginsDockerImageFlag != "" {
+		httpTriggerBinaryName = "http_trigger"
+	}
```

Comment:
curious, is that the same binary or do we actually have 2 separate ones for each type?